### ISSUE TITLE
Doesn't meet accessibility standards for a modal dialog

### ISSUE THREAD

**Issue by tracker1** [dmtrKovalenko/material-ui-pickers#673]
Should add appropriate role and other ARIA attributes...

https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/dialog.html

**Comment by nguyenanduong** [comment]
The button to open the modal dialog does not have aria-label either

**Comment by dmtrKovalenko** [comment]
I fixed missing role=dialog for modal but different aria atrributes like labeled and described should be done on your side.

**Comment by dmtrKovalenko** [comment]
Passed as props
